cypress/integration/t2.js

```diff
@@ -0,0 +1,73 @@
+import T2Page from '../objects/t2Page';
+
+const t2Page = new T2Page();
```

Comment:
you should name this `timeLoggingPage`, T2 is a bad name. Also you should import Login page and use those methods for login